goldmane/proto/api.proto

```diff
@@ -105,6 +105,19 @@ message FilterHintsRequest {
   // - A value greater than zero indicates an absolute time in seconds since the Unix epoch.
   // - A value less than zero indicates a relative number of seconds from "now", as determined by the server.
   int64 start_time_lt = 4;
+
+  // Page specifies the page number to return. It requires that PageSize is also specified in order
+  // to determine page boundaries. Note that pages may change over time as new flow data is collected or expired.
+  // Querying the same page at different points in time may return different results.
+  int64 page = 5;
+
+  // PageSize configures the maximum number of results to return as part of this query.
+  int64 page_size = 6;
+}
+
+message FilterHintsResponse {
+  int64 total = 1;
```

Comment:
Need API documentation comments on this guy.

I expected "total" to be "total number of hints that matched the query" and not "total number of pages of hints that match the query"